tests/mgbench/client.cpp

```diff
@@ -58,6 +58,10 @@ DEFINE_bool(validation, false,
             "Set to true to run client in validation mode."
             "Validation mode works for singe query and returns results for validation"
             "with metadata");
+DEFINE_int32(time_dependent_execution, -1,
+             "Time dependend executins executes the queries for a specified number of minutes."
```

Comment:
Isn't there a better choice in seconds or milliseconds? Maybe even double in seconds, something that maps to `std::chrono::duration<double>` (seconds expressed as `double`)

Pure milliseconds look best to me at the moment 🤔

Reply:
@gitbuda There are two things. First, it is much easier to think in minutes than in seconds and definitely milliseconds, so running some workload for benchmark.py for a constrained time in minutes seems most straightforward from that side. 
On the other hand, using minutes loses granularity, .... ok moving to seconds. they seems to be best for intuition and granurality we need. 😄